app/komponenter/filopplastingfelt/FilopplastningFelt.tsx

```diff
@@ -0,0 +1,61 @@
+import { DownloadIcon } from '@navikt/aksel-icons';
+import { Table } from '@navikt/ds-react';
+import { useState } from 'react';
+import Dropzone from 'react-dropzone';
+
+import css from './filopplastningfelt.module.css';
+
+const FilopplastningFelt = () => {
+  const [filer, settFiler] = useState<File[]>([]);
+
+  const håndterLastOppFil = (nyeFiler: File[]) => {
+    settFiler(gamleFiler => [...gamleFiler, ...nyeFiler]);
+  };
+
+  return (
+    <>
+      <Dropzone onDrop={acceptedFiles => håndterLastOppFil(acceptedFiles)}>
+        {({ getRootProps, getInputProps }) => (
+          <section
+            className={`${css.dokumentasjonKonteiner}`}
+            {...getRootProps()}
+          >
+            <input {...getInputProps()} />
+            <p>
+              <DownloadIcon title="a11y-title" /> Last opp dokumentasjon
+            </p>
+          </section>
+        )}
+      </Dropzone>
+
+      <section>
+        <p>Filer som er lastet opp:</p>
+
+        <Table>
```

Comment:
Legg til en if-sjekk før denne for å ikke vise tabellen dersom ingen filer er lastet opp. F.eks. om statelisten er tom.
<img width="587" alt="image" src="https://github.com/bekk/nav-familie-endringsmelding/assets/46678893/3142b20e-9e60-4029-a716-e3db431c703a">